### ISSUE TITLE
animate scale gives png's a weird shadow in IE's

### ISSUE THREAD

**Issue by BMCouto** [louisremi/jquery.transform.js#23]
Not sure what can be causing this, but transparent png's when scaled get this weird black shadow on the transparent areas. Any idea of what might be?

Thanks!

**Comment by louisremi** [comment]
Hi @BMCouto, which versions of IE are having this problem? Could you send me a screenshot?

Regards,
Lr

**Comment by karazon** [comment]
This issue occurs in IE < 9 in semi-transparent areas when using either translate, scale or rotate.

**Comment by louisremi** [comment]
This sounds like a browser bug and I probably won't be able to fix it.
Can you create a test-case on jsbin.com or jsfiddle.net?

**Comment by BMCouto** [comment]
I believe you are just missing the filters like paulirish has on css3please.com , or not?

**Comment by louisremi** [comment]
This shouldn't be a filter problem, as those black shadows are not visible before using transforms, am I right?

**Comment by BMCouto** [comment]
They are, as karazon says, an IE below 9 shows those shadows using transform.

**Comment by BMCouto** [comment]
@louisremi any news on this?

**Comment by BMCouto** [comment]
@louisremi have you look at this situation, besides this black shadow on png's, the zoom animate move on a weird direction on IE7 and IE8.

**Comment by louisremi** [comment]
@BMCouto I'd like to help you but you never send me any screenshot or test-case demonstrating your problem.

**Comment by dy-dx** [comment]
I am having this issue in Chrome when using translate and scale.
http://i.imgur.com/dUd8b.png

**Comment by louisremi** [comment]
Hi chris,
This looks like a rendering bug lying either in Chrome, your graphic driver our your OS. There's absolutely nothing I can do about it. sorry.

**Comment by dy-dx** [comment]
You are right, it is a rendering bug in the current (v21) version of Chrome. Hopefully they fix it soon! Thanks.
